src/ResourcesAndSupport.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Resources and Support - Crowd Management Platform</title>
+    <link rel="stylesheet" href="styles.css">
+    <script src="scripts.js" defer></script>
+</head>
+<body>
+    <header>
+		<img src="Logo.svg" alt="Crowd Management Platform Logo">
+		<div>
+			<h1>Resources and Support</h1>
+		</div>
+    </header>
+    <main>
+        <section class="guidelines">
```

Comment:
Likewise for this file - React components. There should not be any `.html` files added.